modules/react/tokens/index.ts

```diff
@@ -1,4 +1,15 @@
-import * as canvasColorsWeb from '@workday/canvas-colors-web';
+import {
+  buttonColors,
+  chartingColorOffsets,
+  chartingColors,
+  colors,
+  commonColors,
+  gradients,
+  iconColors,
+  inputColors,
+  statusColors,
+  typeColors,
+} from '@workday/canvas-colors-web';
```

Comment:
It seemed odd to me that `gradients` is the only color key excluded from tokens `canvas` default export. Was this intentional? Was it a mistake? I left it out of the export, but we could add it back without it being a breaking change.

Reply:
E.g. Why is `canvas.chartingColorOffsets` a thing, but not `canvas.gradients`?

Reply:
Yeah, I'd say it's worth adding 👍  Not sure why it got left out